src/argilla_sdk/_api/_fields.py

```diff
@@ -78,19 +82,7 @@ def list(self, dataset_id: UUID) -> List[FieldModel]:
     def _model_from_json(self, response_json: Dict) -> FieldModel:
         response_json["inserted_at"] = self._date_from_iso_format(date=response_json["inserted_at"])
         response_json["updated_at"] = self._date_from_iso_format(date=response_json["updated_at"])
-        return self._get_model_from_response(response_json=response_json)
+        return FieldModel(**response_json)
 
     def _model_from_jsons(self, response_jsons: List[Dict]) -> List[FieldModel]:
         return list(map(self._model_from_json, response_jsons))
-
-    def _get_model_from_response(self, response_json: Dict) -> FieldModel:
```

Comment:
By changing a bit the `FieldModel` definition, we don't need this method anymore